client_example/32_gemm_mx/gemm_mx_fp8.cpp

```diff
@@ -0,0 +1,488 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2024, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
The data range is incorrect, but...

The requirement for copyright year/year range no longer exists.  You can use a simpler format.
```suggestion
// Copyright © Advanced Micro Devices, Inc., or its affiliates.
// SPDX-License-Identifier: MIT
```

Reply:
Thanks John. Changes are made.